windows/install_embedded_pythons.ps1

```diff
@@ -0,0 +1,79 @@
+#
+# Installs embedded python2 and python3 for use in unit testing.
+# Be careful to use unique environment variables indicating location
+# of the python files.
+#
+# We need to provide a location so that the test scripts can find python,
+# but also need to make sure it doesn't confuse the actual builds.
+#
+
+# Uses:
+# EMBEDDED_PYTHON_2_VERSION
+# EMBEDDED_PYTHON_3_VERSION
+
+# this downloads the necessary file
+
+$ErrorActionPreference = 'Stop'
+
+function DownloadFile{
+    param(
+        [Parameter(Mandatory = $true)][string] $TargetFile,
+        [Parameter(Mandatory = $true)][string] $SourceURL
+    )
+    $ErrorActionPreference = 'Stop'
+    $ProgressPreference = 'SilentlyContinue'
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+    Write-Host -ForegroundColor Green "Downloading $SourceUrl to $TargetFile"
+    (New-Object System.Net.WebClient).DownloadFile($SourceURL, $TargetFile)
+}
+
+function DownloadAndExpandTo{
+    param(
+        [Parameter(Mandatory = $true)][string] $TargetDir,
+        [Parameter(Mandatory = $true)][string] $SourceURL
+    )
+    $tmpOutFile = New-TemporaryFile
+
+    DownloadFile -TargetFile $tmpOutFile -SourceURL $SourceURL
+
+    If(!(Test-Path $TargetDir))
+    {
+        md $TargetDir
+    }
+
+    Start-Process "C:\program files\7-Zip\7z.exe" -ArgumentList "x -o${TargetDir} $tmpOutFile" -Wait
+    Remove-Item $tmpOutFile
+}
+
+$py2 = "https://s3.amazonaws.com/dd-agent-omnibus/python-windows-${Env:EMBEDDED_PYTHON_2_VERSION}-amd64.zip"
+$py3 = "https://s3.amazonaws.com/dd-agent-omnibus/python-windows-${Env:EMBEDDED_PYTHON_3_VERSION}-amd64.zip"
+
+if ( $Env:TARGET_ARCH -eq "x86") {
+    $py2 = "https://s3.amazonaws.com/dd-agent-omnibus/python-windows-${Env:EMBEDDED_PYTHON_2_VERSION}-x86.zip"
+    $py3 = "https://s3.amazonaws.com/dd-agent-omnibus/python-windows-${Env:EMBEDDED_PYTHON_3_VERSION}-x86.zip"
+} 
+
+
+$py2Target = "c:\embeddedpy\py2.${Env:EMBEDDED_PYTHON_2_VERSION}"
```

Comment:
Won't this become `c:\embeddedpy\py2.2.7.17` (note double `2.`)?

Reply:
That's indeed the case; the directory name doesn't matter though as long as it's different for the two embedded pythons.